internal/hns/hnsendpoint.go

```diff
@@ -56,6 +65,16 @@ type EndpointResquestResponse struct {
 // HNSEndpointRequest makes a HNS call to modify/query a network endpoint
 func HNSEndpointRequest(method, path, request string) (*HNSEndpoint, error) {
 	endpoint := &HNSEndpoint{}
+	if windowsBuild == osversion.RS1 {
```

Comment:
Wondering; does this apply to RS2 as well, or is that completely obsolete?

Reply:
That is obsolete.  Also, RS2 is not a LTSC or SAC branch.  
Our expectation is customers are on RS1 (server2016) or RS5 (server 2019) or the latest (semi-annual channel) branch.

Reply:
Main reason for asking was if we should be defensive and, e.g. change this to

```go
if windowsBuild < osversion.RS3 {
```

or

```go
if windowsBuild < osversion.RS5 {
```

Reply:
@pradipd Unfortunately.... I think < RS5 would be more prudent. There are at least plenty of docker users still using RS3 and RS4 :/ And when they switch to using containerd as a runtime, I think we need to be more defensive. (That said, hopefully soon we can disable the use of RS3 and RS4 entirely).

Reply:
OK. Sounds good to me.

Reply:
Updated to use

```go
if windowsBuild < osversion.RS5 {
```

And updated the commit message, and comment accordingly PTAL